requirements/base.txt

```diff
@@ -208,28 +207,29 @@ django-extensions==3.1.5 \
 bleach==3.3.0 \
     --hash=sha256:6123ddc1052673e52bab52cdc955bcb57a015264a1c57d37bea2f6b817af0125 \
     --hash=sha256:98b3170739e5e83dd9dc19633f074727ad848cbedb6026708c8ac2d3b697a433
-https://github.com/aleray/mdx_outline/archive/41e22622a17a0d8e410bd921698ec767213a6305.tar.gz#egg=mdx_outline \
```

Comment:
Was there any indication this was actually being held back deliberately, rather than (as is usually the case) being hash-installed while waiting for a later release?

Reply:
The prior commit was from 2013. The one commit newer, which this version bump moves to, is from 2017. I didn't see anything that should hold us back so my guess was this was likely before the project was pushed to pypi.